go.mod

```diff
@@ -47,6 +47,7 @@ require (
 	github.com/h2non/filetype v1.1.3
 	github.com/hashicorp/go-retryablehttp v0.7.4
 	github.com/hashicorp/golang-lru v0.5.1
+	github.com/hashicorp/golang-lru/v2 v2.0.7
```

Comment:
@ahrav i thought we already had an lru cache somewhere? does it just not use this package?

Reply:
https://github.com/trufflesecurity/trufflehog/blob/68f28a0e34f9f95bbfc533595756c6525c7b508f/go.mod#L49C7-L49C7 yep.

Reply:
maybe not v2?